sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -28,31 +28,44 @@ import org.apache.commons.lang3.time.FastDateFormat
 
 import org.apache.spark.sql.internal.SQLConf
 
-sealed trait DateTimeFormatter {
+sealed trait TimestampFormatter {
```

Comment:
Why did we name it `TimestampFormatter`? It has `DateFormatter` as well.

Reply:
Eh, sorry I mean the file name @cloud-fan.

Reply:
will fix it in https://github.com/apache/spark/pull/23329